[HEALTH-356] Update test harness for symbolized keys (#16)
1debit/simplekiq: README.md

A higher priority means that queue will be sampled more often and have a higher chance of running the job.

### Testing

> Instead of requiring `sidekiq/testing`, we should require `simplekiq/testing` in our tests. This has all the functionality of the original sidekiq gem described [here](https://github.com/mperham/sidekiq/wiki/Testing) but also adheres to the symbolized params paradigm established by Simplekiq in tests.

Reviewer: nit: wording

```suggestion
Instead of requiring `sidekiq/testing`, require `simplekiq/testing` in your tests. This has all the functionality of the [original sidekiq gem](https://github.com/mperham/sidekiq/wiki/Testing), but also adheres to the symbolized params paradigm established by Simplekiq in tests.
```
